server.js

```diff
@@ -202,8 +218,9 @@ function newConnection(socket) {
 			}
 
 			io.emit('updatePlayersInGame', table.PlayersList); 
+		} else {
+			console.log("Need to wait for more players.");
```

Comment:
Here we can also add an ( to send the client that we can't start the game and make the client spit out an alert message
```
socket.emit('game_start', false) 
```